atlp-rwanda/cod-fe
# ch-setup-react-project-#181414550

## What does this PR do?
- Set up an empty react project

## Description of task
- The team should create a react project from scratch using Webpack 4, Babel, and the Airbnb linting option.

## How to test this PR?
- install nodejs and **yarn** package manager
- run **yarn install**
- run "npm run "**eslint-fix**" to see eslint working
- then run "**yarn run dev**"
